app/Http/Controllers/Order/ExtendedOrderController.php

```diff
@@ -265,6 +281,50 @@ private function expiryTill($expiry, $expiryTill, $join)
         }
     }
 
+    /**
+     * Searches for Subcription From Date.
+     *
+     * @param string $expiry The Subcription From Date
+     * @param object $join
+     *
+     * @return Query
+     */
+    private function subFrom($till, $from, $join)
+    {
+        if ($from) {
```

Comment:
what happens if from is null? what will it return?
Please try to keep same return type wherever possible

Reply:
Add test cases for this

Reply:
When null, it doesn't filters subscription from field.